docs/UserGuide.md

```diff
@@ -900,7 +912,7 @@ Removes all your entries from EduTrack when you want to start fresh.
 
 ### Saving the data
 
-EduTrack's got your back. Your data is automatically saved on your hard disk after any command that changes the data, so no need to worry about saving manually.
+EduTrack got your back. Your data is automatically saved on your hard disk after any command that changes the data, so no need to worry about saving manually.
```

Comment:
I think with the ('s) is actually grammatically correct. If not, it can be replaced with "EduTrack has got your back"

Reply:
Ohh oops, thanks!!